client/src/main/java/com/kritsit/casetracker/client/CaseTrackerClient.java

```diff
@@ -51,6 +51,8 @@ private void launchUI(Staff user, Stage stage) throws IOException {
                         ui = UserInterfaceFactory.getEditorFrame(); 
                         break;
         }
-        ui.run(user, stage);
+        if(ui != null){
```

Comment:
This null check is needed to clear "Dereferencing possible null pointer." Realistically, ui should never be null at that point. It makes FindBugs happy though.